app/handlers/trusted_parameters_launch.rb

```diff
@@ -0,0 +1,45 @@
+class TrustedParametersLaunch
+
+  attr_reader :user_state, :attrs
+
+  lev_handler
+
+  protected
+
+  def authorized?
+    true
+  end
+
+  def setup
+    @attrs = request.session['trusted']
+    @user_state = options[:user_state]
```

Comment:
this is totally fine here... but just fyi normally stuff in `setup` is that which you need to be available when `authorized?` is called.  Otherwise it can just go into `handle`.